src/components/TokensPanel.vue

```diff
@@ -43,6 +44,11 @@ async function loadNextTokens() {
 
 const limit = computed(() => process.client && isDesktop() ? 10 : 3)
 
+await useAsyncData(async() => {
```

Comment:
Other usages of `useAsyncData` does not have to return true. EG `fetchOracleDetails`. Why here is it different?

Reply:
They always do (or at least they should for ensuring correct deduplication of requests between SSR and CSR). In the fetchOracleDetails return true is included in the fetch function, but all of our simple fetch functions return Promise<void> so I just added return true in the useAsyncData to keep it consistent. Proof:

![image](https://github.com/aeternity/aescan/assets/46789227/e0f66f95-44fa-4a10-9ab0-2f1bbf272369)

Reply:
Understood. Thanks for explanation